ecommerce/extensions/partner/strategy.py

```diff
@@ -33,8 +35,18 @@ def availability_policy(self, product, stockrecord):
         return availability.Unavailable()
 
 
+class TaxStrategyMixin(strategy.FixedRateTax):
+    """
+    Tax strategy for e-commerce pricing.
+    Defaults to 0% tax, but allows setting a fixed tax rate through Django settings.
+    """
+
+    @property
+    def rate(self):
```

Comment:
Here I changed it to @property instead of the class variable to make easier testing with override_settings, also Test this in the UI and it is working fine